server/models/db/school.js

```diff
@@ -0,0 +1,30 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+
+const UserTypes = require('../../constants/userTypes.json');
+const Tables = require('../../constants/tables.json');
+
+const WorkerDetails = require('../data/Worker');
+
+// This method returns the corresponding worker ids given school ID. If the school ID is incorrect, it will return null. 
+// If either the tableName or Object are null or undefined, an error will be thrown.
```

Comment:
Hi Simran,
I would recommend removing this comment as it does not pertain to your specific method - it looks to be accidentally copied from another method.

Reply:
Hi Amy! Thanks for the comment! That was a great catch! I've removed the comment from my code! 

Thanks!